packages/E2ETest/wdio/test/controlStyle.spec.ts

```diff
@@ -15,25 +15,25 @@ beforeAll(() => {
 describe('ControlStyleTest', () => {
   /* Test case #1: Controls style with regular border */
 
-  it('ControlStyleTestWithRegularBorder', () => {
-    const result = ControlStyleTestPage.getTreeDumpResult();
+  it('ControlStyleTestWithRegularBorder', async () => {
+    const result = await ControlStyleTestPage.getTreeDumpResult();
     assert(
       result,
       '#1. Dump comparison for Control style with regular border!'
     );
   });
 
   /* Test case #2: Click button once, update controls style and round border*/
-  it('ControlStyleTestWithRoundBorder', () => {
+  it('ControlStyleTestWithRoundBorder', async () => {
     ControlStyleTestPage.toggleControlBorder();
-    const result = ControlStyleTestPage.getTreeDumpResult();
+    const result = await ControlStyleTestPage.getTreeDumpResult();
     assert(result, '#2. Dump comparison for Control style with round border!');
   });
 
   /* Test case #3: Click button one more, return to #1*/
-  it('ControlStyleTestWithRegularBorder #2', () => {
+  it('ControlStyleTestWithRegularBorder #2', async () => {
     ControlStyleTestPage.toggleControlBorder();
-    const result = ControlStyleTestPage.getTreeDumpResult();
+    const result = await ControlStyleTestPage.getTreeDumpResult();
```

Comment:
It looks like fixing the broken 100ms wait leads to this no longer working. Not sure if we should remove the wait or what is going on here quite yet.

Reply:
@asklar: Maybe he knows...

---
In reply to: [487580996](https://github.com/microsoft/react-native-windows/pull/5980#discussion_r487580996) [](ancestors = 487580996)